en/messages.json

```diff
@@ -123,6 +123,12 @@
   "removeThisContainerConfirmation": {
     "message": "Are you sure you want to remove this Container?"
   },
+  "clearContainerStoragePanelTitle": {
+    "message": "Clear Container Storage"
+  },
+  "clearContainerStorageConfirmation": {
+    "message": "Are you sure you want to clear cookies and storage for this Container?"
```

Comment:
This uses the same pattern as previous strings, but it doesn't make a lot of sense for `container` to be uppercase. It would be worth discussing with content about it.